docs/source/users/index.md

```diff
@@ -177,6 +177,43 @@ To compose a message, type it in the text box at the bottom of the chat interfac
     alt='Screen shot of an example "Hello world" message sent to Jupyternaut, who responds with "Hello world, how are you today?"'
     class="screenshot" />
 
+### Usage with SageMaker Endpoints
+
+Jupyter AI supports language models hosted on SageMaker Endpoints that use JSON
+APIs. The first step is to authenticate with AWS via the `boto3` SDK and have
+the credentials stored in the `default` profile.  Guidance on how to do this can
+be found in the
+[`boto3` documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html).
+
+When selecting the SageMaker Endpoints provider in the settings panel, you will
+see the following interface:
+
+<img src="../_static/chat-sagemaker-endpoints.png"
+    height="50%"
+    width="50%"
```

Comment:
```suggestion
    height="307"
    width="399"
```

The `50%` values would refer to the containing box, not to the image itself

Reply:
I don't see a reason for preferring this over `50%`. This has no impact on the network bandwidth either, since it's still fetching the entire image at render and then scaling it to the dimensions specified.